permissions/plugin-support-core.yml

```diff
@@ -8,4 +8,4 @@ developers:
 - "schristou"
 - "stephenconnolly"
 - "batmat"
-
+- "owood"
```

Comment:
Good news (sort of): When I created the initial batch of files, none had a trailing newline, which makes almost all diffs look like crap now.